lib/matplotlib/patches.py

```diff
@@ -720,22 +733,18 @@ def __init__(self, xy, width, height, angle=0.0, **kwargs):
         Parameters
         ----------
         xy : (float, float)
-            The rectangle extends from xy[0] to xy[0] + width in
-            x-direction and from xy[1] to xy[1] + height in y-direction.
+            The anchor point.
         width : float
-            Rectangle width
+            Rectangle width.
         height : float
-            Rectangle height
+            Rectangle height.
         angle : float, default: 0
             Rotation in degrees anti-clockwise about *xy*.
-        fill : bool, default: True
-            Whether to fill the rectangle.
 
-        Notes
-        -----
-        Valid keyword arguments are:
-
-        %(Patch)s
+        Other Parameters
+        ----------------
+        **kwargs : `Patch` properties
```

Comment:
I think it is unhappy about this reference?


```
/home/circleci/project/lib/matplotlib/table.py:docstring of matplotlib.table.Cell:123: WARNING: py:obj reference target not found: Patch
/home/circleci/project/lib/matplotlib/table.py:docstring of matplotlib.table.CustomCell:90: WARNING: py:obj reference target not found: Patch
```

but the location makes no sense...

Reply:
Well it does make sense: Wonders of inherited docstrings.

`Cell` inherits from rectangle. And likely (I don‘t have code access right now) it reuses the docstring. However, Patch is not in the module `table`. The solution is to use `` `.Patch` ``. I will fix this later (or anybody can push that change).

Reply:
> I think it is unhappy about this reference?
> 
> 
> 
> 
> 
> ```
> 
> /home/circleci/project/lib/matplotlib/table.py:docstring of matplotlib.table.Cell:123: WARNING: py:obj reference target not found: Patch
> 
> /home/circleci/project/lib/matplotlib/table.py:docstring of matplotlib.table.CustomCell:90: WARNING: py:obj reference target not found: Patch
> 
> ```
> 
> 
> 
> but the location makes no sense...


Well it does make sense: Wonders of inherited docstrings.

`Cell` inherits from rectangle. And likely (I don‘t have code access right now) it reuses the docstring. However, Patch is not in the module `table`. The solution is to use `` `.Patch` ``. I will fix this later (or anybody can push that change).